src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const countDays = days * 40;
+
+  if (days >= 7) {
+    return countDays - 50;
+  }
+
+  if (days >= 3) {
+    return countDays - 20;
+  }
+
+  if (days < 3) {
+    return countDays;
+  }
```

Comment:
The condition `if (days < 3)` is redundant because if the previous conditions (`days >= 7` and `days >= 3`) are not met, it is implied that `days` must be less than 3. You can replace this condition with a simple `else` statement.